### PR TITLE
Question bulk service

### LINKED ISSUE #139: Backend: CreateQuestionsBulk service to speed up  create evaluation

### PR DESCRIPTION
**Pull request by Christdej**
Closes #139

### CODE REVIEW

File: scripts/Pipfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+pandas = "*"
+
+[requires]
+python_version = "3.6"
```
Comment (nicholasdalhaug) [inline scripts/Pipfile:12]: Is it mentioned anywhere how to use this? It seems like it is Pipenv, and I have never used it before. Maybe say in our readme how to install. Is it `pipenv install`?

File: scripts/create_json/create_json.py [hunk 2]
```diff
@@ -1,14 +1,34 @@
...
+    ]
+]
+df = df.rename(
+    columns={
+        "PS": "Barrier",
+        "Question": "Text",
+        "Support Notes": "SupportNotes",
+        "Responsible/ Related to: (eller noe i den duren)": "Organization",
+    }
+)
```
Comment (nicholasdalhaug) [inline scripts/create_json/create_json.py:31]: If we rename first, can we get a prettier pick of columns? 

```suggestion
df = df.rename(
    columns={
        "PS": "Barrier",
        "Question": "Text",
        "Support Notes": "SupportNotes",
        "Responsible/ Related to: (eller noe i den duren)": "Organization",
    }
)
df = df[
    [
        "Barrier",
        "Text",
        "SupportNotes",
        "Organization",
    ]
]
```

Comment (Christdej) [reply scripts/create_json/create_json.py]: Agreed and done =)

File: scripts/Pipfile (hunk 1 above)
Comment (Christdej) [reply scripts/Pipfile]: Removed pipenv file

**Review by nicholasdalhaug — APPROVED** [review]

**Review by ReedOnly — COMMENTED** [review]
see comment

File: scripts/create_json/create_json.py [hunk 3]
```diff
@@ -1,14 +1,33 @@
...
 
 data_path = os.getcwd()
 df = pd.read_excel(
-    os.path.join(data_path, "Q&As.xlsx", ),
-    sheet_name="Check lists")
+    os.path.join(
+        data_path,
+        "Q&As.xlsx",
+    ),
+    sheet_name="Checklists FINAL",
```
[the remark this answers is not in the document]
Comment (Christdej) [reply scripts/create_json/create_json.py:12]: Yes it was, but I suspect we have to change these every time we get a new xlsx file from them

**Review by ReedOnly — APPROVED** [review]

**Pull request merged**